src/frameworks/ez.md

```diff
@@ -18,4 +18,129 @@ In particular, see:
+
+## Local Development with eZ Platform 2.x and later
+
+eZ Systems provide a tool called [eZ Launchpad](https://ezsystems.github.io/launchpad/) for local development on top of a Docker stack. It improves Developer eXperience and reduces complexity for common actions by simplifying your interactions with Docker containers. eZ Launchpad is ready to work with Platform.sh.
+
+### eZ Launchpad installation
+
+eZ Launchpad's approach is to stay as decoupled as possible from your development machine and your remote hosting whether you are Linux or Mac OSX. To install run:
+
+```bash
+curl -LSs https://ezsystems.github.io/launchpad/install_curl.bash | bash
+```
+ 
+Then you can start to use it to initialize your eZ Platform project on top Docker.
+
+```bash
+~/ez init
+```
+
+or create the Docker stack based on an existing project
+
+```bash
+git clone yourproject.git application
+cd application
+~/ez create
+```
+
+You will find more details on the [eZ Launchpad documentation](https://ezsystems.github.io/launchpad/).
+
+At this time you will have a working eZ Platform application with many services including Varnish, Solr, Redis etc.
+
+### Platform.sh integration
+
+To generate the key files for Platform.sh (`.platform.app.yaml` and `.platform`) run:
+
+```bash
+~/ez platformsh:setup
+```
+
+eZ Launchpad will generate the files for you and you are then totally free to fine tune them.
+
+#### Solr specificity
+
+Solr is fully functional with eZ Launchpad but its auto-configuration for Platform.sh is not currently managed. You will have to set it up manually following the current documenation here: https://github.com/ezsystems/ezplatform/blob/master/.platform/services.yaml#L37.
+
+Actions needed are:
+
+ * generare the Solr configuration thanks to the script provided by eZ Systems
+ * put the result in the `.platform` at the root of your project
+ * add the service in the `.platform/services.yaml`
+ * add the relationship in the `.platform.app..yaml`
+
+#### Environment variables (optional)
+
+eZ Launchpad allows you to define environment variables in the `provisioning/dev/docker-compose.yml` file. You may use that to set [Platform.sh variables](/development/variables.md) to match Platform.sh environments so that you can keep your environment behavior in sync.
+
+Such variables have to be set in the `engine` container.
+
+```yaml
+# provisioning/dev/docker-compose.yml
+engine:
+    environment:
+        - ASIMPLEVARIABLE=avalue
+        - PLATFORM_RELATIONSHIPS=A_BASE64_ENCODED_VALUE
+```
+
+### Local development with Platform.sh
+
+Thanks to eZ Launchpad we are able to be work 100% locally: [untethered](/gettingstarted/local/untethered.md). We have the whole project working offline on our local machine.
```

Comment:
Avoid the use of "we" to refer to the user.  The user is "you".  "We" means Platform.sh, Inc.  (Or whatever the French equivalent of "Inc" is.)

Reply:
thanks @Crell for all the typos (including this one) you are fixing. I do very well know the difference between `you` and `we`.